.ci/collectApk.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo "Collect apps..."
```

Comment:
I'd rather set `addons.artifacts.paths` in `.travis.yml` instead of having this script.